src/components/Playlists.js

```diff
@@ -1,171 +1,144 @@
-                    Footer
-                    </Typography>
-                    <Typography variant="subtitle1" align="center" color="textSecondary" component="p">
-                    Something here to give the footer a purpose!
-                    </Typography>
-                    <Copyright />
-                </footer>
-                {/* End footer */}
-    </React.Fragment>
-
-        );
-    }
+class Playlists extends React.Component {
+  constructor(props) {
+    super(props);
+  }
 
+  render() {
+    const { classes, theme } = this.props;
+    return (
+      <React.Fragment>
+        <CssBaseline />
+        <main>
+          {/* Hero unit */}
+          <div className={classes.heroContent}>
+            <Container maxWidth="sm">
+              <Typography
+                component="h1"
+                variant="h2"
+                align="center"
+                color="textPrimary"
+                gutterBottom
+              >
+                Playlists
+              </Typography>
+              <Typography
+                variant="h5"
+                align="center"
+                color="textSecondary"
+                paragraph
+              >
+                Something short and leading about the collection below—its
+                contents, the creator, etc. Make it short and sweet, but not too
+                short so folks don&apos;t simply skip over it entirely.
+              </Typography>
+            </Container>
+          </div>
+          <Container className={classes.cardGrid} maxWidth="md">
+            {/* End hero unit */}
+            <Grid container spacing={4}>
+              {cards.map((card) => (
+                <Grid item key={card} xs={12} sm={6} md={4}>
+                  <Card className={classes.card}>
+                    <CardMedia
+                      className={classes.cardMedia}
+                      image="https://source.unsplash.com/random"
+                      title="Image title"
+                    />
+                    <CardContent className={classes.cardContent}>
+                      <Typography gutterBottom variant="h5" component="h2">
+                        Playlist Name.
+                      </Typography>
+                      <Typography>
+                        Playlist description.
+                      </Typography>
+                    </CardContent>
+                    <CardActions>
+                      <Button size="small" color="primary">
+                        View
+                      </Button>
+                      <Button size="small" color="primary">
+                        Edit
+                      </Button>
+                    </CardActions>
+                  </Card>
+                </Grid>
+              ))}
+            </Grid>
+          </Container>
+        </main>
+      </React.Fragment>
+    );
+  }
 }
 
-const mapStateToProps = state => {
-    return {
-        
-    }
-}
+const mapStateToProps = (state) => {
+  return {};
```

Comment:
I told Mikayla about this too but if you wrap the curly brackets after arrow with brackets you don't need the return statement ie:
const something = (param) => ({
     someKey: someValue,
});

just a little neater imo but not required :D